core/arm/demoDashboard.json

```diff
@@ -150,6 +150,9 @@
           "apiVersion": "[variables('web-site-version')]",
           "dependsOn": [
             "[variables('web-site-id')]",
+            "[concat('Microsoft.Web/sites/', variables('web-site-name'), '/config/appsettings')]",
```

Comment:
this is dangerous: as a result of the appsettings update, the appservice can restart while installing the extension.
to be on a safe side, i'd limit this to the two updates below and make sure to give it a few tries.